tslint.json

```diff
@@ -78,6 +78,7 @@
             true,
             "allow-null-check"
         ],
+        "trailing-comma": true,
```

Comment:
this branch was originally made before I had to deal with the tslint update for `for...of`. I made a branch off master to deal with fixing merge conflicts from the `thing.FSP` PR and I think at that time master already had an updated tslint.json. Not sure why it didn't show removing `no-trailing-commas` as well.
Either that happened, or I had to switch `no-trailing-comma` to `trailing-comma` for the linter to be happy....I can't remember which happened.